docs/tools/vdb_table/data/rockset.json

```diff
@@ -14,139 +14,139 @@
     "comment": ""
   },
   "license": {
-    "value": "",
-    "source_url": "",
+    "value": "Proprietary",
+    "source_url": "https://rockset.com/legal/terms-of-service/",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      "not known"
+      "c++"
     ],
     "source_url": "",
     "comment": ""
   },
   "github_stars": 0,
-  "vector_launch_year": 0,
+  "vector_launch_year": 2023,
   "metadata_filter": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/sofia099/search_autocomplete",
     "comment": ""
   },
   "facets": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.rockset.com/documentation/reference/aggregate-functions",
     "comment": ""
   },
   "geo_search": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.rockset.com/documentation/reference/geographic-functions",
     "comment": ""
   },
   "multi_vec": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "sparse_vectors": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "bm25": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "full_text": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.rockset.com/documentation/reference/text-search-functions",
     "comment": ""
   },
   "embeddings_text": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.rockset.com/documentation/reference/user-defined-functions",
     "comment": ""
   },
   "embeddings_image": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.rockset.com/documentation/reference/user-defined-functions",
```

Comment:
The ability to run user-defined functions counts as "partial" support for these 2 columns. Full support would be for ready to use embedding modules (see examples in the table).

![Screenshot 2024-02-02 at 1 25 03 PM](https://github.com/superlinked/VectorHub/assets/105786647/0aeac021-15fb-4115-a5af-0e615ae37a18)

I'll switch these and merge the PR. You can comment here if there is further clarification.